packages/db/prisma/schema.prisma

```diff
@@ -1,8 +1,8 @@
 // This is your Prisma schema file,
 // learn more about it in the docs: https://pris.ly/d/prisma-schema
 datasource db {
-  provider = "postgresql"
-  url      = env("DATABASE_URL")
+  provider = "sqlite"
+  url      = "file:./admin.local.db"
```

Comment:
is there a particular reason for this? I think we should make to example based on postgres because there is where we get the most features

Reply:
I'd love to have an example that I get actually test :)